.travis.yml

```diff
@@ -27,20 +27,12 @@ script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   - if [[ `uname` = "Linux" ]]; then TESTCMD="xvfb-run julia"; else TESTCMD="julia"; fi
   - $TESTCMD -e 'using Pkg; Pkg.build(); Pkg.test(coverage=true)'
-after_success:
-  - julia -e 'using Pkg;
-              import Gtk;
-              cd(joinpath(dirname(pathof(Gtk))));
-              Pkg.add("Coverage");
-              using Coverage;
-              Coveralls.submit(Coveralls.process_folder())'
-  # update the documentation
-  - julia -e 'using Pkg;
-              ps=Pkg.PackageSpec(name="Documenter", version="0.19");
-              Pkg.add(ps);
-              Pkg.pin(ps)'
-  - $TESTCMD -e 'using Pkg;
-                 import Gtk;
-                 cd(joinpath(dirname(pathof(Gtk))));
-                 ENV["DOCUMENTER_DEBUG"] = "true";
-                 include(joinpath("docs", "make.jl"))'
+jobs:
+  include:
+    - stage: "Documentation"
+      julia: 1.0
+      os: linux
+      script:
+        - julia --project=docs/ -e 'import Pkg; Pkg.instantiate(); Pkg.develop(Pkg.PackageSpec(path=pwd()))'
```

Comment:
```suggestion
        - if [[ `uname` = "Linux" ]]; then TESTCMD="xvfb-run julia"; else TESTCMD="julia"; fi
        - $TESTCMD --project=docs/ -e 'import Pkg; Pkg.instantiate(); Pkg.develop(Pkg.PackageSpec(path=pwd()))'
        - $TESTCMD --project=docs/ docs/make.jl
```

Reply:
To build doc we precise to use linux. Is that `$TESTCMD` really needed?

Reply:
I don't think so. @vtjnash what where the intentions?